roles/nova-data/handlers/main.yml

```diff
@@ -11,3 +11,13 @@
   service: name={{ item.name }} state=restarted must_exist=false
   with_items:
     - "{{ nova.services.nova_compute }}"
+
+- name: restart multipath-tools
```

Comment:
See comments below on using variables for service name and reduce this is one task.